platform-sdk/swirlds-cli/src/main/java/com/swirlds/cli/logging/CssRuleSetFactory.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.cli.logging;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Utility class for turning CSS rules into a single CSS string, for inclusion in an HTML page.
+ */
+public class CssRuleSetFactory {
```

Comment:
Right now I generate the JTR report by manually forming strings. We may want to use these utility classes for both the log generator and my reporting tool. I wonder if it would make sense to pull these into a different package, something like `com.swirlds.platform.gui.html`, for example.

Reply:
I created an issue to do this, and will work after this merges https://github.com/hashgraph/hedera-services/issues/8333

I want to avoid major restructurings, as I have some additional features branching of the current changes, and don't want a difficult merge